### PR TITLE
[Merged by Bors] - chore(Order/ConditionallyCompleteLattice/Basic): ConditionallyCompleteLinearOrderBot extends OrderBot

### PR DESCRIPTION
**Pull request by vihdzp**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by vihdzp** [comment]
!bench

File: Mathlib/Order/ConditionallyCompleteLattice/Basic.lean [hunk 1]
```diff
@@ -219,17 +219,10 @@ complete linear orders, we prefix `sInf` and `sSup` by a `c` everywhere. The sam
...
-  Bot α where
-  /-- `⊥` is the least element -/
-  bot_le : ∀ x : α, ⊥ ≤ x
-  /-- The supremum of the empty set is `⊥` -/
+  OrderBot α where
+  /-- The supremum of the empty set is special-cased to `⊥` -/
   csSup_empty : sSup ∅ = ⊥
 
--- see Note [lower instance priority]
-instance (priority := 100) ConditionallyCompleteLinearOrderBot.toOrderBot
```
Comment (eric-wieser) [inline Mathlib/Order/ConditionallyCompleteLattice/Basic.lean:229]: Can you attach this instance priority? I think there is a syntax for this using `attribute [instance`, hopefully a search can find it.

**Comment by eric-wieser** [comment]
!bench

**Comment by vihdzp** [comment]
!bench

**Comment by vihdzp** [comment]
What's going on? Is the speedcenter down?

**Comment by vihdzp** [comment]
!bench

**Comment by vihdzp** [comment]
!bench

File: Mathlib/Order/ConditionallyCompleteLattice/Basic.lean [hunk 2]
```diff
@@ -219,16 +219,12 @@ complete linear orders, we prefix `sInf` and `sSup` by a `c` everywhere. The sam
 hold in both worlds, sometimes with additional assumptions of nonemptiness or
 boundedness. -/
 class ConditionallyCompleteLinearOrderBot (α : Type*) extends ConditionallyCompleteLinearOrder α,
-  Bot α where
-  /-- `⊥` is the least element -/
-  bot_le : ∀ x : α, ⊥ ≤ x
-  /-- The supremum of the empty set is `⊥` -/
+  OrderBot α where
```
Comment (eric-wieser) [inline Mathlib/Order/ConditionallyCompleteLattice/Basic.lean:222]: To distinguish this from the fields below, just like we distinguish hypotheses from proofs:
```suggestion
    OrderBot α where
```

bors d+

**Pull request closed without merging**